src/pages/en/tutorial/1-setup/1.mdx

```diff
@@ -26,15 +26,22 @@ You will use a **command line (terminal)** to create your Astro project and to r
 
 You can access the command line through a local terminal program for your operating system. Common applications include **Terminal** (MacOS/Linux), **Command Prompt** (Windows), and **Termux** (Android). One of these will probably already be on your machine. 
 
-For Astro to run on your system, you will also need to have [**Node.js**](https://nodejs.org/en/) installed. To check to see whether you already have a compatible version (`v16.12.0` or higher) installed: 
+### Node.js
 
-1. Launch your terminal and type the following into the command line:
+For Astro to run on your system, you also need to have [**Node.js**](https://nodejs.org/en/) of version `v16.12.0` or later.
```

Comment:
```suggestion
For Astro to run on your system, you will also need to have [**Node.js**](https://nodejs.org/en/) installed. 
```
I agree that Node is significant enough that it can have its own heading! I'm just reworking to avoid the phrase "Node of version" which is a little unusual here in the context of the rest of our text.

I've also saved mention of the version number for the next line.

Reply:
I disagree. Node is basically the only technical prerequisite to run Astro (terminal and code editor are just tools of choice), so making readers aware of the minimum compatible version is **crucial**, and this information should be provided as early as possible. We can further think (and agree, for sure) about rewording, but I strongly believe that details about the version must be there.